### ISSUE TITLE
Custom UI for setting up CMDT

### ISSUE THREAD

**Issue by JodieM** [SFDO-Community/Salesforce-Indicators#64]
This is one for the way future... 

And you can modify CMDT outside the Salesforce UI in multiple ways now (I like Enabler4Excel best). So it is very much a nice to have. 

Maybe start with a proof of concept to do it via a Flow?

**Comment by tschug** [comment]
Yes - if we could do this with Flow, that would be a great start!

We could look to make it very user-friendly with the [UnofficialSF's Icon Picker](https://unofficialsf.com/icon-picker-for-flow-screens-and-custom-property-editors/) too!

**Comment by tschug** [comment]
This is very similar to the modal idea #88  - except it also has the "quick" links to CMDT in Setup.

Another "future" version would be having a specific Form LWC that uses the Metadata API with a callout to perform CMDT DML (See DLRS)

**Comment by JodieM** [comment]
Closed as this is in the latest release now.

**Comment by JodieM** [comment]
Re-opening to track the new work that was done in the Chicago sprint.
